### PR TITLE
PR-cleanup

### PR DESCRIPTION
**Pull request by ZenaMel**

### CODE REVIEW

File: code-base/src/main.py [hunk 1]
```diff
@@ -1,58 +1,49 @@
...
         print("Invalid flags!")
 
 
-
 # Helper Functions
 # ------------------------------------------------------------------
 
 """First screen the user will see"""
-def titleScreen():
+def title_screen():
```
Comment (meta-boy) [inline code-base/src/main.py:26]: [Type hinting](https://docs.python.org/3/library/typing.html) for your functions. can be pretty helpful.

File: code-base/src/main.py [hunk 2]
```diff
@@ -1,58 +1,49 @@
...
-    projectName = input("Project Name: ") 
+def ask_for_project_name():
+    project_name = input("Project Name: ") 
     print("Creating Project...") 
-    if basic_project(projectName):
+    if basic_project(project_name):
         print("Finished Creating Project") 
         print("[Summary of Project]") 
-        print("Project Name: " + projectName) 
+        print("Project Name: " + project_name)
```
Comment (meta-boy) [inline code-base/src/main.py]: You might wanna use `click.echo()` instead of  `print()`

**Review by meta-boy — CHANGES_REQUESTED** [review]

File: code-base/src/main.py (hunk 2 above)
Comment (ZenaMel) [reply code-base/src/main.py]: yah right! Thanks.

File: code-base/src/main.py (hunk 1 above)
Comment (ZenaMel) [reply code-base/src/main.py:26]: nice, reminds me of Haskell. Is it actually useful though? Python is dynamically typed after all.

**Comment by ZenaMel** [comment]
Stale in favor of #60

File: code-base/src/main.py (hunk 1 above)
Comment (meta-boy) [reply code-base/src/main.py:26]: > nice, reminds me of Haskell. Is it actually useful though? Python is dynamically typed after all.

Due to the dynamic nature of Python, inferring, or checking the type of an object being used is especially hard. With type hinting, you hint the type of the object(s) you're using. If you are more curious you can check this presentation here: https://www.youtube.com/watch?v=2wDvzy6Hgxg

**Pull request closed without merging**